client/dashboard/components/settings/general/store-address.js

```diff
@@ -200,6 +215,7 @@ export function StoreAddress( props ) {
 			<SelectControl
 				label={ __( 'Country / Region', 'woocommerce-admin' ) }
 				required
+				autoComplete="new-password" // disable autocomplete and autofill
```

Comment:
This seems to be the only way to have the chrome autofill now show up. Chrome ignores the use of `autoComplete='off'` 😢

Reply:
Nice workaround!